examples/stocks/lib/stock_menu.dart

```diff
@@ -6,12 +6,14 @@ part of stocks;
 
 enum _MenuItems { autorefresh, autorefreshCheckbox, add, remove }
 
+const double _kMenuMargin = 16.0; // 24.0 on tablet
+
 Future showStockMenu(NavigatorState navigator, { bool autorefresh, ValueChanged onAutorefreshChanged }) async {
   switch (await showMenu(
     navigator: navigator,
     position: new MenuPosition(
-      right: sky.view.paddingRight,
-      top: sky.view.paddingTop
+      right: sky.view.paddingRight + _kMenuMargin,
+      top: sky.view.paddingTop + _kMenuMargin
```

Comment:
This shouldn't be pushed to the users. It's a material design spec margin, it should be implemented in the widgets. The author should just have to give the vague location that the menu should appear at ("top right", or "near this button", or some such) and we should apply the margins automatically.